From 78987190d457d21f78324ba18734e1962ad2c45c Mon Sep 17 00:00:00 2001
From: Lijun Fang <fanglijun3@huawei.com>
Date: Thu, 23 Dec 2021 21:31:56 +0800
Subject: [PATCH] arm64/ascend: Enable CONFIG_ASCEND_DVPP_MMAP
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: 78987190d457d21f78324ba18734e1962ad2c45c


ascend inclusion
category: feature
bugzilla: https://gitee.com/openeuler/kernel/issues/I4M24Q
CVE: NA

-------------------------------------------------

Enable CONFIG_ASCEND_DVPP_MMAP by default y.

Signed-off-by: Lijun Fang <fanglijun3@huawei.com>
Reviewed-by: Weilong Chen <chenweilong@huawei.com>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 arch/arm64/configs/openeuler_defconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/configs/openeuler_defconfig b/arch/arm64/configs/openeuler_defconfig
index 589364f741de..17bc8750bba7 100644
--- a/arch/arm64/configs/openeuler_defconfig
+++ b/arch/arm64/configs/openeuler_defconfig
@@ -475,6 +475,8 @@ CONFIG_ARM64_PSEUDO_NMI=y
 CONFIG_RELOCATABLE=y
 CONFIG_RANDOMIZE_BASE=y
 CONFIG_RANDOMIZE_MODULE_REGION_FULL=y
+CONFIG_ASCEND_FEATURES=y
+CONFIG_ASCEND_DVPP_MMAP=y
 # end of Kernel Features
 
 #
-- 
2.26.2

